aiida_registry/fetch_metadata.py

```diff
@@ -31,6 +31,17 @@
 
 GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
 
+def get_previous_plugins_info():
```

Comment:
do you really need to get it from github?

can't you get it from the local git clone?